src/main/scala/smoke/Server.scala

```diff
@@ -106,11 +106,17 @@ trait Server {
 
   val logDateFormat = new SimpleDateFormat("dd/MMM/yyyy:HH:mm:ss Z")
 
+  val logQueryParams = config.getBoolean("log-query-params")
```

Comment:
It's in reference.conf. The value will always be present, like "error-log-verbose" so no need for a default.